### ISSUE TITLE
Deleting pipes with a barrel conveyance doesn't delete the barrel conveyance

### ISSUE THREAD

**Issue by mjuber-kci** [leeensminger/DelDOT-NPDES-Field-Tool#146]
If you delete all conveyances within the barrel conveyance, the barrel conveyance does not get deleted. The barrel conveyance is left in the database with 0 pipes.

![capture](https://cloud.githubusercontent.com/assets/16919958/14115024/62743100-f5a7-11e5-870f-c92bf5efb22b.JPG)

**Comment by leeensminger** [comment]
@mjuber-kci If you subsequently delete the Barrel Conveyance, does that work?

**Comment by mjuber-kci** [comment]
Yes you can delete the Barrel Conveyance afterwards. However, if the current user did not originally inventory the barrel conveyance, they could delete all the conveyances individually without knowing that those conveyances were within a barrel conveyance. Therefore they could easily delete the conveyances and not the barrel conveyance. Is it possible to have a message pop up when you try to delete a conveyance that is part of a barrel conveyance, such as, "This Conveyance is part of a Barrel Conveyance, do you still wish to delete this conveyance?". Or possibly make it easily visible/identifiable from the conveyance form if the conveyance is part of a barrel conveyance.
